lesson-3/README.md

```diff
@@ -12,15 +12,73 @@
 
 ### Setup ETCD Operator 
 
-Clone etcd-operator repository:
-```bash
-git clone git@github.com:coreos/etcd-operator.git
-```
-
 Install RBAC rules (cluster Roles and RoleBindings):
 ```bash 
-etcd-operator/example/rbac/create_role.sh
+# Create ClusterRole object
+echo | kubectl create -f - <<EOF
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: etcd-operator
+rules:
+- apiGroups:
+  - etcd.database.coreos.com
+  resources:
+  - etcdclusters
+  - etcdbackups
+  - etcdrestores
+  verbs:
+  - "*"
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - "*"
+- apiGroups:
+  - ""
+  resources:
+  - pods
+  - services
+  - endpoints
+  - persistentvolumeclaims
+  - events
+  verbs:
+  - "*"
+- apiGroups:
+  - apps
+  resources:
+  - deployments
+  verbs:
+  - "*"
+# The following permissions can be removed if not using S3 backup and TLS
+- apiGroups:
+  - ""
+  resources:
+  - secrets
+  verbs:
+  - get
+EOF
 
+# Create ClusterRoleBinding object
+echo | kubectl create -f - <<EOF
```

Comment:
don't do this we expect people to be able to call kubectl commands and we ant them to think not copy paste.

Reply:
@dbecvarik so what should I do?